src/tests/stress/pedsim/Ped_util/Agent_passive.enc

```diff
@@ -0,0 +1,92 @@
+module Agent_passive
+import Global_funs
+unsafe class Agent
```

Comment:
Added tiny comment at the top of the file, also comment before `next` as that's the only method with more then 5 lines outside of the constructor.